Doctrine/EventSubscriber.php

```diff
@@ -64,7 +66,12 @@ public function persistObject(DataEvent $event)
         }
 
         $objectManager->persist($event->getData());
-        $objectManager->flush();
+        try {
+            $objectManager->flush();
+        } catch (\Exception $e) {
```

Comment:
But I only want my event (`FLUSH_ERROR`) to be dispatched when an error occurred, so `finally` seems to don't be the right method for it, isn't it ?